app/models/review.rb

```diff
@@ -1,3 +1,5 @@
 class Review < ApplicationRecord
   belongs_to :booking
+
+  validates :content, presence: true
```

Comment:
should we require the presence of the booking too?

Reply:
since the review is generated by the system itself i can't see when it could be added a review without that info. but wee can add that with no problems

Reply:
Yes, you will never have a review without a booking_id. That's for sure. The same happens with user_id on toilet. 
It will never cause us a problem having it with those validations or not having. 
And we dont need them to work, that's right.